deltachat-ios/Controller/ClosedAccountErrorViewController.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  ClosedAccountErrorViewController.swift
+//  deltachat-ios
+//
+//  Created by bb on 05.07.22.
+//  Copyright © 2022 Jonas Reinsch. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import DcCore
+
+class ClosedAccountErrorView: UIView {
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Error: Account is closed"
+        label.textColor = DcColors.grayTextColor
+        label.textAlignment = .center
+        label.numberOfLines = 1
+        label.font = UIFont.systemFont(ofSize: 24, weight: .bold)
```

Comment:
we are setting the constraints manually in the code, therefore all UIElements need to have the flag
`translatesAutoresizingMaskIntoConstraints` set to false